packages/esm-patient-registration-app/src/patient-registration/patient-registration.test.tsx

```diff
@@ -379,7 +383,7 @@ describe('patient registration component', () => {
       expect(givenNameInput.value).toBe('John');
       expect(familyNameInput.value).toBe('Wilson');
       expect(middleNameInput.value).toBeFalsy();
-      expect(dateOfBirthInput.value).toBe('4/4/1972');
+      expect(dateOfBirthInput.value).toBe('04/04/1972');
```

Comment:
So this change was necessary because of Carbon. In the new date picker, we changed the way we trickle down the `value` prop. Currently (in registration), this prop is being passed to the `DatePickerInput` this changed and it's now being consumed by the `DatePicker`. Somehow the new way overrides the value's formatting.

Source of inspiration? @ibacher :
> As long as those minimal changes are addressed and we have some assurance that changing from DatePicker -> OpenmrsDatePicker will be tested with both the Ge'ez and Gregorian calendars, we can go ahead and merge the earlier version of this.

([check out thread](https://github.com/openmrs/openmrs-esm-core/pull/746#issuecomment-1701420744))

Reply:
Plus according to the Carbon docs, the `DatePicker` is the property consumer: https://react.carbondesignsystem.com/?path=/docs/components-datepicker--overview

Reply:
This change might be obviated by an update to the library if the logic change upstream.